### PR TITLE
U-Touch: Add Czech strings

### PR DESCRIPTION
**Pull request by SuperLamic**

### CODE REVIEW

**Comment by AlvaroBrey** [comment]
Thank you!

**Comment by SuperLamic** [comment]
You're welcome! :P

File: overlay/packages/apps/CMParts/res/values-cs/strings.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+    <string name="hardware_keys_action">Akce</string>
+    <string name="hardware_keys_long_tap_key_title">Akce dlouhého podržení</string>
+    <string name="hardware_keys_swipe_left_key_title">Akce přesunutí z prava do leva</string>
+    <string name="hardware_keys_swipe_right_key_title">Akce přesunutí z leva do prava</string>
```
Comment (mbalous) [inline overlay/packages/apps/CMParts/res/values-cs/strings.xml:6]: No such words as "leva" or "prava" exists.

**Review by mbalous — COMMENTED** [review]
Small mistake in translation.

File: overlay/packages/apps/CMParts/res/values-cs/strings.xml (hunk 1 above)
Comment (mbalous) [reply overlay/packages/apps/CMParts/res/values-cs/strings.xml:6]: It should be "zleva" or "zprava" instead...

**Pull request merged**